src/contexts/__tests__/current-product.test.tsx

```diff
@@ -1,6 +1,6 @@
-import { ReactNode } from 'react'
+import { ReactNode, Component } from 'react'
 import { render, screen } from '@testing-library/react'
-import { renderHook } from '@testing-library/react-hooks'
```

Comment:
This library got included in `@testing-library/react` as part of the changes to support react 18, so making the switch.